presto-hive/src/main/java/io/prestosql/plugin/hive/HiveSessionProperties.java

```diff
@@ -110,6 +111,14 @@ public static InsertExistingPartitionsBehavior valueOf(String value, boolean imm
 
             return enumValue;
         }
+
+        public static InsertExistingPartitionsBehavior getDefaultValue(HiveConfig hiveConfig)
+        {
+            boolean immutable = hiveConfig.isImmutablePartitions();
+            Optional<String> defaultValue = hiveConfig.getInsertExistingPartitionsBehavior();
+            if (defaultValue.isEmpty()) {return immutable ? ERROR : APPEND;}
```

Comment:
```suggestion
            if (defaultValue.isEmpty()) {
                return immutable ? ERROR : APPEND;
            }
```

We do not fold if statements. The only exception is empty method like:
```
void method() {}
```
Otherwise we always put curly brackets into separate lines.

Reply:
I hope this PR will be accepted: https://github.com/airlift/codestyle/pull/12